### ISSUE TITLE
spotify player does not play in iOS simulator

### ISSUE THREAD

**Issue by raji16** [spotify/ios-sdk#934]
-  Installed the latest spotify demo iOS sdk
- Clicked on login to spotify button
- It asked for login credentials, entered the login credentials and it took me to the player screen
- But the player is not getting played on iOS simulator

**Comment by arirawr** [comment]
Hi @raji16! Does playback work when you test using a physical device?

Are you using a Spotify Premium account? Playback is only supported for Spotify Premium users.

If not, are you getting any error messages?

**Comment by raji16** [comment]
Yes , Playback work on physical device, but when i run the same on iOS simulator, playback delegates are not getting called.
I am using a spotify premium account .
I dont get any error messages for the same.
For testing purpose , i have to everytime connect my device since the playback delegate is not getting called on iOS simulator.
